packages/react-composites/package.json

```diff
@@ -16,7 +16,7 @@
     "build:e2e:chat": "webpack-cli build --config ./tests/browser/chat/app/webpack.config.js --mode=production --env production",
     "test": "jest",
     "test:coverage": "npm run test -- --coverage",
-    "test:e2e:chat": "playwright test tests.browser.chat",
+    "test:e2e:chat": "playwright test --workers=1 tests.browser.chat",
```

Comment:
this looks good, but just double checking this wasn't a param added for testing something locally

Reply:
Unfortunately we can't run these tests in parallel with any amount of robustness. The thread creation will fail intermittently because we're creating multiple threads with the same connection string in quick succession.